modules/nw-cluster-mtu-change.adoc

```diff
@@ -316,37 +311,28 @@ machineconfiguration.openshift.io/state: Done
 +
 Verify that the following statements are true:
 +
---
- * The value of `machineconfiguration.openshift.io/state` field is `Done`.
- * The value of the `machineconfiguration.openshift.io/currentConfig` field is equal to the value of the `machineconfiguration.openshift.io/desiredConfig` field.
---
+* The value of `machineconfiguration.openshift.io/state` field is `Done`.
+* The value of the `machineconfiguration.openshift.io/currentConfig` field is equal to the value of the `machineconfiguration.openshift.io/desiredConfig` field.
 
 .. To confirm that the machine config is correct, enter the following command:
 +
 [source,terminal]
 ----
-$ oc get machineconfig <config_name> -o yaml | grep path:
+$ oc get machineconfig <config_name> -o yaml | grep path: <1>
 ----
-+
-where `<config_name>` is the name of the machine config from the `machineconfiguration.openshift.io/currentConfig` field.
+<1> Where `<config_name>` is the name of the machine config from the `machineconfiguration.openshift.io/currentConfig` field.
 +
 If the machine config is successfully deployed, the previous output contains the `/etc/NetworkManager/conf.d/99-<interface>-mtu.conf` file path and the `ExecStart=/usr/local/bin/mtu-migration.sh` line.
 endif::local-zone,wavelength-zone,post-aws-zones,outposts[]
 
-. To finalize the MTU migration, enter the following command for the OVN-Kubernetes network plugin:
+. To finalize the MTU migration, enter the following command for the OVN-Kubernetes network plugin. Replace `<mtu>` with the new cluster network MTU that you specified with `<overlay_to>`.
 +
 [source,terminal]
-+
 ----
 $ oc patch Network.operator.openshift.io cluster --type=merge --patch \
   '{"spec": { "migration": null, "defaultNetwork":{ "ovnKubernetesConfig": { "mtu": <mtu> }}}}'
 ----
-+
---
-where:
-
-`<mtu>`:: Specifies the new cluster network MTU that you specified with `<overlay_to>`.
---
+Where `<mtu>` specifies the new cluster network MTU that you specified with `<overlay_to>`.
```

Comment:
suggestion to turn this into a code callout

Reply:
:heavy_check_mark: